src/engraving/dom/engravingitem.cpp

```diff
@@ -1276,8 +1276,12 @@ void EngravingItem::relinkPropertiesToMaster(PropertyGroup propGroup)
         }
         const PropertyValue masterValue = masterElement->getProperty(propertyId);
         const PropertyFlags masterFlags = masterElement->propertyFlags(propertyId);
-        setProperty(propertyId, masterValue);
-        setPropertyFlags(propertyId, masterFlags);
+        if (getProperty(propertyId) != masterValue) {
```

Comment:
Can I conclude from this change that if you call `setProperty(propertyId, masterValue);` while `getProperty(propertyId) == masterValue` changes the Offset property? If yes, that would surprise me a bit. Could you explain why that would happen (and shouldn't we fix that)?

Reply:
Because one of the properties for text items is the TextStyle, which itself collects a bunch of different properties, among which is also the offset (different text style have different default offsets etc*). Without this check, toggling the text re-link was causing the TextStyle property to be unnecessarily updated and therefore also the offset.

*BTW, the fact that we use the offset property (i.e. the same property used for _user_ offset) to determine the _default_ position of an item (meaning that there exists such thing as a "default offset") make me absolutely furious: it will always cause these kinds of issues (I've seen many similar ones before). It _will_ be changed, when I get the chance.